visualization/src/pcl_visualizer.cpp

```diff
@@ -892,6 +892,8 @@ bool
 pcl::visualization::PCLVisualizer::removeActorFromRenderer (const vtkSmartPointer<vtkLODActor> &actor, int viewport)
 {
   vtkLODActor* actor_to_remove = vtkLODActor::SafeDownCast (actor);
+  if (!actor_to_remove)
```

Comment:
`actor` is a pointer to `vtkLODActor` by definition, I don't see how this cast may fail...